linux_os/guide/system/software/system-tools/package_rear_installed/rule.yml

```diff
@@ -0,0 +1,25 @@
+documentation_complete: true
+
+prodtype: fedora,ol7,ol8,rhel6,rhel7,rhel8
+
+title: 'Install rear Package'
+
+description: |-
+    {{{ describe_package_install(package="rear") }}}
+rationale: |-
```

Comment:
Sory for nitpick, but for better readability please put new line before rationale.